start-page/beta/index.html

```diff
@@ -2,39 +2,169 @@
+              </svg>
+            </a>
+          </section>
+
+          <section id="bug">
+            <h2 class="section-heading bug">
+              <svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" height="24" width="24">
+                <path d="M12 0a5.866 5.866 0 0 0-5.867 5.867c0 .279.064.543.178.778A8.9 8.9 0 0 0 4.3 9H1V6.434a.5.5 0 0 0-.5-.5.5.5 0 0 0-.5.5V9.5a.5.5 0 0 0 .5.5h3.291a8.928 8.928 0 0 0-.725 3.533V14H.5a.5.5 0 0 0-.5.5.5.5 0 0 0 .5.5h2.566v.066A8.93 8.93 0 0 0 3.75 18.5H.5a.5.5 0 0 0-.5.5v3.066a.5.5 0 0 0 .5.5.5.5 0 0 0 .5-.5V19.5h3.242A8.921 8.921 0 0 0 12 24a8.92 8.92 0 0 0 7.758-4.5H23v2.566a.5.5 0 0 0 .5.5.5.5 0 0 0 .5-.5V19a.5.5 0 0 0-.5-.5h-3.25a8.923 8.923 0 0 0 .684-3.434V15H23.5a.5.5 0 0 0 .5-.5.5.5 0 0 0-.5-.5h-2.566v-.467c0-1.256-.26-2.45-.725-3.533H23.5a.5.5 0 0 0 .5-.5V6.434a.5.5 0 0 0-.5-.5.5.5 0 0 0-.5.5V9h-3.3a8.935 8.935 0 0 0-2.01-2.355 1.78 1.78 0 0 0 .177-.778A5.866 5.866 0 0 0 12 0Zm0 1a4.846 4.846 0 0 1 4.867 4.867c0 .286-.071.416-.244.563-.173.146-.488.284-.928.377-.42.088-.946.136-1.54.162-.613.018-1.313.021-2.155.021-.635 0-1.226 0-1.768-.008h-.03c-.742-.021-1.395-.07-1.897-.175-.44-.093-.755-.23-.928-.377-.173-.147-.244-.277-.244-.563A4.846 4.846 0 0 1 12 1ZM6.951 7.457c.484.225.981.41 1.697.465.365.028.77.043 1.196.053.67.027 1.395.025 2.156.025 1.487 0 2.84.008 3.9-.215l.037-.008a6.58 6.58 0 0 0 1.124-.304c1.737 1.452 2.873 3.603 2.873 6.06v1.533A7.905 7.905 0 0 1 12 23a7.905 7.905 0 0 1-7.934-7.934v-1.533c0-2.464 1.139-4.623 2.885-6.076zM9.5 9A1.5 1.5 0 0 0 8 10.5 1.5 1.5 0 0 0 9.5 12a1.5 1.5 0 0 0 1.5-1.5A1.5 1.5 0 0 0 9.5 9Zm5 0a1.5 1.5 0 0 0-1.5 1.5 1.5 1.5 0 0 0 1.5 1.5 1.5 1.5 0 0 0 1.5-1.5A1.5 1.5 0 0 0 14.5 9Zm-5 4A1.5 1.5 0 0 0 8 14.5 1.5 1.5 0 0 0 9.5 16a1.5 1.5 0 0 0 1.5-1.5A1.5 1.5 0 0 0 9.5 13Zm5 0a1.5 1.5 0 0 0-1.5 1.5 1.5 1.5 0 0 0 1.5 1.5 1.5 1.5 0 0 0 1.5-1.5 1.5 1.5 0 0 0-1.5-1.5Zm-5 4A1.5 1.5 0 0 0 8 18.5 1.5 1.5 0 0 0 9.5 20a1.5 1.5 0 0 0 1.5-1.5A1.5 1.5 0 0 0 9.5 17Zm5 0a1.5 1.5 0 0 0-1.5 1.5 1.5 1.5 0 0 0 1.5 1.5 1.5 1.5 0 0 0 1.5-1.5 1.5 1.5 0 0 0-1.5-1.5Z"/>
+                <path d="M12 .5a5.355 5.355 0 0 1 5.367 5.366C17.367 8.839 14.973 7.5 12 7.5S6.633 8.84 6.633 5.866A5.355 5.355 0 0 1 12 .5Z" opacity=".2"/>
+              </svg>
+              Report
+            </h2>
+            <p>{{ _('Think you&apos;ve found a new problem or rough edge? Create a bug report so we can make sure it gets solved, resulting in a better Thunderbird experience for all.') }}</p>
```

Comment:
We use unicode `’` for apostrophes elsewhere, they do look different than the entity version so lets stay consistent for now. It's a different character from `'` so it does not mess with the single quotes around the string.